build.sh

```diff
@@ -184,7 +185,7 @@ case $OSName in
 esac
 __MSBuildBuildArch=x64
 __BuildType=Debug
-__CMakeArgs=DEBUG
```

Comment:
No. I just removed a redundant variable '_CmakeArgs' which always has the same value as '__BuildType'. You can still pass no flavor parameter to build.sh and '__BuildType' will be initialized to 'Debug' and passed onto cmake.